src/nightly.js

```diff
@@ -128,7 +128,7 @@ module.exports = function*(argv) {
     yield npmpublish.publishTag(options);
 
     //unpublish old nightly
-    options.version = oldNightlyVersion;
+    options.version = oldNightlyVersions;
     yield npmpublish.unpublish(options);
```

Comment:
I'd rather just call this once per repo, specifying id and version separately for each repo. This way you don't need to change anything in `src/npm-publish.js`